File: mycollab-services/src/main/java/com/esofthead/mycollab/module/project/domain/criteria/FollowingTicketSearchCriteria.java
package com.esofthead.mycollab.module.project.domain.criteria;

import com.esofthead.mycollab.common.domain.criteria.MonitorSearchCriteria;
import com.esofthead.mycollab.core.arguments.StringSearchField;

public class FollowingTicketSearchCriteria extends MonitorSearchCriteria {
	private static final long serialVersionUID = 1L;

	
	private StringSearchField summary;


	public StringSearchField getSummary() {
		return summary;
	}


	public void setSummary(StringSearchField summary) {
		this.summary = summary;
	}
}

File: mycollab-services/src/main/scala/com/mycollab/module/project/domain/criteria/FollowingTicketSearchCriteria.scala
package com.mycollab.module.project.domain.criteria

import com.mycollab.common.domain.criteria.MonitorSearchCriteria
import com.mycollab.db.arguments.StringSearchField

import scala.beans.BeanProperty

/**
  * @author MyCollab Ltd
  * @since 5.2.10
  */
class FollowingTicketSearchCriteria extends MonitorSearchCriteria {
  @BeanProperty var name: StringSearchField = _
}
